packages/kbn-config-schema/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "outDir": "target",
     "rootDir": "src",
     "sourceMap": true,
-    "sourceRoot": "../../../../../packages/kbn-config-schema/src",
+    "sourceRoot": "../../../../packages/kbn-config-schema/src",
```

Comment:
Is this because it's from the `bazel-bin/packages/kbn-config-schema/target` directory and not the root of the package?

Reply:
typescript is configured to not follow symlinks so it will consider `node_modules/@kbn/config-schema/target` 😃

Reply:
what happened here was that previously we had `target/out` and `target/types` and as part of the migration into bazel I put everything on `target` but forgot to rearrange the source root setting